<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>芒果超市</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/Download/Chrome/芒果.ico</normaloff>D:/Download/Chrome/芒果.ico</iconset>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1581</width>
     <height>811</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_commodity">
    <widget class="QTableWidget" name="tableWidget_commodity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1491</width>
       <height>781</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>商品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>我的</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_shop">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>11</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>购物车</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
